astropy/io/ascii/cparser.pyx

```diff
@@ -738,6 +741,9 @@ cdef class CParser:
     def set_names(self, names):
         self.names = names
 
+    def get_header_names(self):
+        return self.header_names
+
```

Comment:
What the benefit of making this a function instead of just accessing `self.header_names`?